src/ConnectedMode/Suppressions/ServerIssuesStore.cs

```diff
@@ -66,12 +74,30 @@ public void UpdateIssue(string issueKey, bool isResolved)
 
             if (issue == null)
             {
+                logger.LogVerbose(Resources.Store_UpdateIssue_NoMatch, issueKey);
                 return;
             }
 
-            issue.IsResolved = isResolved;
+            if (issue.IsResolved == isResolved)
```

Comment:
This is a change of behaviour - the event is only raised if the property has actually changed.

I guess I could have made this change first in a separate PR but I'd already made most of the logging changes by then, and I decided it was a small enough change that it wasn't worth the extra work.

Reply:
Yea makes sense!